src/components/reusables/reaction/Reaction.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react'
```

Comment:
Please comment on the imports and also move the `export default` to last line

Reply:
done